buildSrc/src/main/java/Dependencies.kt

```diff
@@ -19,10 +19,10 @@ object Dependencies {
 
     object Compose {
       const val activity = "androidx.activity:activity-compose:1.3.1"
-      const val foundation = "androidx.compose.foundation:foundation:1.0.1"
-      const val material = "androidx.compose.material:material:1.0.1"
-      const val tooling = "androidx.compose.ui:ui-tooling:1.0.1"
-      const val ui = "androidx.compose.ui:ui:1.0.1"
+      const val foundation = "androidx.compose.foundation:foundation:1.1.0-rc01"
+      const val material = "androidx.compose.material:material:1.1.0-rc01"
+      const val tooling = "androidx.compose.ui:ui-tooling:1.1.0-rc01"
+      const val ui = "androidx.compose.ui:ui:1.1.0-rc01"
```

Comment:
Note that this is the latest version for these libraries, but there's an `rc02` for the compiler plugin with Kotlin 1.6.10 support. We'll use the same versions internally very soon.

Reply:
Could you [file an issue](https://github.com/square/workflow-kotlin/issues/new) tracking this, and add a comment here?

Reply:
https://github.com/square/workflow-kotlin/issues/619